packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -154,6 +159,26 @@ export class VaCheckbox {
     if (this.enableAnalytics) this.fireAnalyticsEvent();
   };
 
+  /**
+   * For a11y, input.indeterminate must be set with JavaScript, there is no HTML attribute for this.
+   */
+  private updateIndeterminateInput() {
+    const input = this.el.shadowRoot.querySelector('input');
+    if (this.indeterminate && !this.checked) {
+      input.indeterminate = true;
```

Comment:
We are adding this indeterminate attribute manually with JavaScript based on [Ryan's a11y findings here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3365#issuecomment-2426922846). When I tested with VO, I confirmed that this is what announces "mixed" as the checkbox state when it's active.